build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -17,9 +17,10 @@ def is_number?(value)
   true if Integer(value) rescue false
 end
 
-def substituteMultiline(multilineLogging, new_contents)
-    if !multilineLogging.nil? && multilineLogging.to_s.downcase == "true"
+def substituteMultiline(multilineLogging, stacktraceLanguages, new_contents)
+    if !multilineLogging.nil? && multilineLogging.to_s.downcase == "true" && !stacktraceLanguages.nil? && !stacktraceLanguages.empty?
```

Comment:
This will be breaking change for existing customers since existing customers doesnt have languages specified, right? For keep existing customers behavior , If customer doesn't specify the stack traces languages and we should  default to all languages ?

Reply:
we will. for existing customers stacktraceLanguages will be populated to all 4 languages in tomlparser.rb